.changelog/4294.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed the logic for retrieving the latest Docker image tag in both the old validate and the new.
```

Comment:
Fixed an issue where the latest docker image validator didn't retrieve the latest docker image version as a fix suggestion.